frontend/components/nav.js

```diff
@@ -1,45 +1,48 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { Link } from 'react-router';
+import shortid from 'shortid';
 
 const propTypes = {
-  username: React.PropTypes.oneOfType([
-    React.PropTypes.string,
-    React.PropTypes.object
-  ])
+  username: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.object,
+  ]),
 };
 
-const getSecondaryNav = (username) => {
-  const links = getSecondaryNavLinks(username);
-
-  return links.map((link, index) => {
-    return <li key={index}>{link}</li>;
-  });
+const defaultProps = {
+  username: null,
 };
 
-const getSecondaryNavLinks = (username) => {
+function getSecondaryNavLinks(username) {
   if (username) {
     return [
       <Link to="/sites">{username}</Link>,
-      <a href="https://federalist-docs.18f.gov" target="_blank">Documentation</a>,
-      <a href="https://github.com/18F/federalist/issues/new" target="_blank">File Issue</a>,
+      <a href="https://federalist-docs.18f.gov" target="_blank" rel="noopener noreferrer">Documentation</a>,
+      <a href="https://github.com/18F/federalist/issues/new" target="_blank" rel="noopener noreferrer">File Issue</a>,
       <a href="mailto:federalist-support@gsa.gov">Get Help</a>,
       <a href="/logout">Log out</a>,
-    ]
-  } else {
-    return [
-      <a href="https://federalist-docs.18f.gov" target="_blank">Documentation</a>,
-      <a href="https://github.com/18F/federalist/issues/new" target="_blank">Contact Us</a>,
-      <a href="/auth/github">Log in</a>,
-    ]
+    ];
   }
+  return [
+    <a href="https://federalist-docs.18f.gov" target="_blank" rel="noopener noreferrer">Documentation</a>,
+    <a href="https://github.com/18F/federalist/issues/new" target="_blank" rel="noopener noreferrer">Contact Us</a>,
+    <a href="/auth/github">Log in</a>,
+  ];
 }
 
+function getSecondaryNav(username) {
+  const links = getSecondaryNavLinks(username);
+  return links.map(link => <li key={shortid.generate()}>{link}</li>);
```

Comment:
using `shortid` to generate unique keys for the links, which was something the linter caught. Alternatively, we could just include the surrounding `<li>...</li>` tags in the arrays of anchors up in `getSecondaryNavLinks`.